article/app/pages/StoryHtmlPage.scala

```diff
@@ -27,6 +27,13 @@ object StoryHtmlPage {
     override def IE9CriticalCss: Html = stylesheetLink(s"stylesheets/ie9.$ContentCSSFile.css")
   }
 
+  def atomStyles(implicit page: Page): Seq[Html] = Page.getContent(page) 
+    .flatMap(_.atoms)
+    .map(_.atomTypes.collect { 
+      case (atomType, true) => stylesheetLink(s"stylesheets/atom-$atomType-article-index.css")
```

Comment:
is this async?

why go with this rather than the webpack approach?

Reply:
Because I don't like having a JS -> CSS dependency that far down in the stack, esp. on mobile. The `LINK` uses the `media` attribute trick, so yes this is async. I realise it is a JS -> CSS but it is the same all the other stylesheets are going through, so I'm fine with that.